iothub/device/tests/Transport/Mqtt/MqttTransportHandlerTests.cs

```diff
@@ -426,23 +430,24 @@ public async Task MqttTransportHandlerSendTwinGetAsyncTimesOut()
         // Tests_SRS_CSHARP_MQTT_TRANSPORT_18_030: If the response contains a success code, `SendTwinPatchAsync` shall return success to the caller.
         // Tests_SRS_CSHARP_MQTT_TRANSPORT_18_035: `SendTwinPatchAsync` shall shall open the transport if this method is called when the transport is not open.
         [TestMethod]
-        public async Task MqttTransportHandlerSendTwinPatchAsyncHappyPath()
+        [DataRow(TwinJsonString1)]
+        [DataRow(TwinJsonString2)]
+        public async Task MqttTransportHandlerSendTwinPatchAsyncHappyPath(string twinJson)
```

Comment:
By calling `SendTwinPatchAsync()` on L458, it can validate `JsonConvert.SerializeObject()` working properly with date-formatted string containing trailing zeros.